src/components/seeds/database/index.tsx

```diff
@@ -452,13 +461,21 @@ export default function Database(props: DatabaseProps): JSX.Element {
         };
 
         const populateFieldOptions = async () => {
-          const singleAndMultiChoiceFields = filterSelectFields(searchColumns);
-          const allValues = await SeedBankService.searchFieldValues(
-            singleAndMultiChoiceFields,
-            {},
-            selectedOrganization.id
+          // Since the seed bank service doesn't return values for project IDs, we need to merge the values in
+          const singleAndMultiChoiceFields = filterSelectFields(
+            searchColumns.filter((column) => column !== 'project_name')
           );
 
+          const allValues =
```

Comment:
this can be null, add a guard around line 473?

Reply:
I am changing it to `{}` if it is `null` on the next line. Would you rather I leave it and check for `null` or is it OK as-is? Currently `allValues` will never be `null`.

Reply:
I missed the `|| {}` my bad.. this is totally fine.